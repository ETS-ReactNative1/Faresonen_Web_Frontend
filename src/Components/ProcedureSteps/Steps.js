import React, { Component } from 'react';
import { Icon, Step } from 'semantic-ui-react';

class Steps extends Component {
	render() {
		return (
			<Step.Group>
			    <Step completed>
			      <Icon name='checkmark' />
			      <Step.Content>
			        <Step.Title>Liga valgt</Step.Title>
			        <Step.Description>Du har valgt {this.props.league}</Step.Description>
			      </Step.Content>
			    </Step>

			    <Step active>
			      <Icon name='numbered list' />
			      <Step.Content>
			        <Step.Title>Velg runde</Step.Title>
			        <Step.Description>Hvilken runde gjelder lefsen for?</Step.Description>
			      </Step.Content>
			    </Step>

			    <Step>
			      <Icon name='print' />
			      <Step.Content>
			        <Step.Title>Print</Step.Title>
			        <Step.Description>Ctrl + P for Ã¥ printe lefsen</Step.Description>
			      </Step.Content>
			    </Step>

			    <Step>
			      <Icon name='cocktail' />
			      <Step.Content>
			        <Step.Title>Bra jobbet</Step.Title>
			        <Step.Description>Ta deg en velfortjent pause!</Step.Description>
			      </Step.Content>
			    </Step>
			</Step.Group>
		)
	}
}

export default Steps;