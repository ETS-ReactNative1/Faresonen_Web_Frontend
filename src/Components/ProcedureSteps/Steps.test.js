import React from "react";
import renderer from "react-test-renderer";
import Steps from "./Steps";

describe("<Steps />", () => {
  it("Should render properly", () => {
    const tree = renderer.create(<Steps league="Eliteserien" />);
    expect(tree).toMatchSnapshot();
  });
});
