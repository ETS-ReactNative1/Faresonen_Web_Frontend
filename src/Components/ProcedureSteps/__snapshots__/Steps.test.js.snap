// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Steps /> Should render properly 1`] = `
<div
  className="ui steps no-print"
>
  <div
    className="completed step"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="checkmark icon"
      onClick={[Function]}
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Liga valgt
      </div>
      <div
        className="description"
      >
        Du har valgt
        Eliteserien
      </div>
    </div>
  </div>
  <div
    className="active step"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="numbered list icon"
      onClick={[Function]}
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Velg runde
      </div>
      <div
        className="description"
      >
        Hvilken runde gjelder lefsen for?
      </div>
    </div>
  </div>
  <div
    className="disabled step"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="print icon"
      onClick={[Function]}
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Print
      </div>
      <div
        className="description"
      >
        Ctrl + P for Ã¥ printe lefsen
      </div>
    </div>
  </div>
  <div
    className="disabled step"
    onClick={[Function]}
  >
    <i
      aria-hidden="true"
      className="cocktail icon"
      onClick={[Function]}
    />
    <div
      className="content"
    >
      <div
        className="title"
      >
        Bra jobbet
      </div>
      <div
        className="description"
      >
        Ta deg en velfortjent pause!
      </div>
    </div>
  </div>
</div>
`;
